src/test-utils/dom/collection-preferences/sticky-columns-preference.ts

```diff
@@ -10,12 +10,12 @@ export default class StickyColumnsPreferenceWrapper extends ComponentWrapper {
   static lastRootSelector = styles['sticky-columns-last'];
 
   findTitle(): ElementWrapper {
-    return this.findComponent(`.${styles[`sticky-columns-form-field`]}`, FormFieldWrapper)!.findLabel()!;
+    return this.findComponent(`.${styles['sticky-columns-form-field']}`, FormFieldWrapper)!.findLabel()!;
```

Comment:
Why is this needed? What happens with the extractor when a template string used?

Reply:
This change is not required. The templates strings like these ones are handled just fine. If a template string includes an identifier it is then replaced with a wildcard, e.g. `flash-type-*` to match all properties starting from `flash-type-`.

Reply:
Let's just throw an error saying that dynamic property access is not allowed and simplify everything

Reply:
I think it works well so far. An alternative could be to update the source code of the test-utils, for example, instead of `styles[`flash-type-${type}`]` we can do:

```
function getFlashStyleByType(type) {
  switch (type) {
    case 'error':
      return styles.error;
    case 'in-progress':
      return styles['in-progress'];
    // ...
  }
}
```

But I would rather keep the current solution with the extractor.

Reply:
Do we have any need for this case today? If not, why do you need to think about it?

Reply:
Yes, there are 22 selectors matched this way:
```
"awsui_content-display-description_tc96w",
"awsui_content-display-option-content_tc96w",
"awsui_content-display-option-list_tc96w",
"awsui_content-display-option_tc96w",
"awsui_content-display-title_tc96w",
"awsui_visible-content-group-label_tc96w",
"awsui_visible-content-group_tc96w",
"awsui_visible-content-groups_tc96w",
"awsui_visible-content-option-label_tc96w",
"awsui_visible-content-option_tc96w",
"awsui_visible-content-title_tc96w",
"awsui_visible-content-toggle_tc96w",
"awsui_flash-type-error_1q84n",
"awsui_flash-type-in-progress_1q84n",
"awsui_flash-type-info_1q84n",
"awsui_flash-type-success_1q84n",
"awsui_flash-type-warning_1q84n",
"awsui_result-container-error_11huc",
"awsui_result-container-success_11huc",
"awsui_navigation-link-active_1xupv",
"awsui_navigation-link-disabled_1xupv",
"awsui_navigation-link-label_1xupv",
```